### ISSUE TITLE
ssdlite-mobilenet_v2  Increase input size from 300 to 900? loss dont convergence

### ISSUE THREAD

**Issue by JaosonMa** [tensorflow/models#5951]
ssdlite_mobilenet_v2_coco.config
i changed 
model {
  ssd {
    num_classes: 185
    box_coder {
      faster_rcnn_box_coder {
        y_scale: 10.0
        x_scale: 10.0
        height_scale: 5.0
        width_scale: 5.0
      }
    }
    matcher {
      argmax_matcher {
        matched_threshold: 0.5
        unmatched_threshold: 0.5
        ignore_thresholds: false
        negatives_lower_than_unmatched: true
        force_match_for_each_row: true
      }
    }
    similarity_calculator {
      iou_similarity {
      }
    }
    anchor_generator {
      ssd_anchor_generator {
        num_layers: 6
        min_scale: 0.2
        max_scale: 0.95
        aspect_ratios: 1.0
        aspect_ratios: 2.0
        aspect_ratios: 0.5
        aspect_ratios: 3.0
        aspect_ratios: 0.3333
      }
    }
    image_resizer {
      fixed_shape_resizer {
        height: 900  # Increase input size from 300 to 900
        width: 900 # 900
      }
    }
...

____________________
i Increase input size from 300 to 900 ,because i want to detect samll object in the image,
but , loss not convergence. i don/t know why?

INFO:tensorflow:global step 441: loss = 37.9318 (0.351 sec/step)
I1223 15:40:34.394506 140216558724928 tf_logging.py:115] global step 441: loss = 37.9318 (0.351 sec/step)
INFO:tensorflow:global step 442: loss = 79.1737 (0.333 sec/step)
I1223 15:40:34.729128 140216558724928 tf_logging.py:115] global step 442: loss = 79.1737 (0.333 sec/step)
INFO:tensorflow:global step 443: loss = 29.9729 (0.290 sec/step)
I1223 15:40:35.027728 140216558724928 tf_logging.py:115] global step 443: loss = 29.9729 (0.290 sec/step)
INFO:tensorflow:global step 444: loss = 34.6217 (0.298 sec/step)
I1223 15:40:35.327668 140216558724928 tf_logging.py:115] global step 444: loss = 34.6217 (0.298 sec/step)
INFO:tensorflow:global step 445: loss = 78.8400 (0.310 sec/step)
I1223 15:40:35.639422 140216558724928 tf_logging.py:115] global step 445: loss = 78.8400 (0.310 sec/step)
INFO:tensorflow:global step 446: loss = 22.9451 (0.296 sec/step)
I1223 15:40:35.936698 140216558724928 tf_logging.py:115] global step 446: loss = 22.9451 (0.296 sec/step)
INFO:tensorflow:global step 447: loss = 77.7216 (0.276 sec/step)
I1223 15:40:36.214318 140216558724928 tf_logging.py:115] global step 447: loss = 77.7216 (0.276 sec/step)
INFO:tensorflow:global step 448: loss = 38.1520 (0.361 sec/step)
I1223 15:40:36.577134 140216558724928 tf_logging.py:115] global step 448: loss = 38.1520 (0.361 sec/step)
INFO:tensorflow:global step 449: loss = 26.1242 (0.296 sec/step)
I1223 15:40:36.874693 140216558724928 tf_logging.py:115] global step 449: loss = 26.1242 (0.296 sec/step)
INFO:tensorflow:global step 450: loss = 21.5703 (0.345 sec/step)
I1223 15:40:37.221611 140216558724928 tf_logging.py:115] global step 450: loss = 21.5703 (0.345 sec/step)
INFO:tensorflow:global step 451: loss = 149.9978 (0.327 sec/step)
I1223 15:40:37.550590 140216558724928 tf_logging.py:115] global step 451: loss = 149.9978 (0.327 sec/step)

loss continue jump ,

how can i trained the 900*900 image with ssdlite_mobilenet_v2?
@drpngx 
@tensorflow-jenkins

**Comment by drpngx** [comment]
Right, the loss doesn't seem right. Try reducing the learning rate?

**Comment by JaosonMa** [comment]
finally ,i notes this:
  #fine_tune_checkpoint: "../ssdlite_mobilenet_v2_coco_2018_05_09/model.ckpt"
  #fine_tune_checkpoint_type:  "detection"
and keep training, 
after 1500 steps, the loss start convergence, 


INFO:tensorflow:global step 138528: loss = 5.4548 (0.501 sec/step)
I1224 11:06:41.304660 140647853295424 tf_logging.py:115] global step 138528: loss = 5.4548 (0.501 sec/step)
INFO:tensorflow:global step 138529: loss = 6.8238 (0.467 sec/step)
I1224 11:06:41.772933 140647853295424 tf_logging.py:115] global step 138529: loss = 6.8238 (0.467 sec/step)
INFO:tensorflow:global step 138530: loss = 5.2648 (0.469 sec/step)
I1224 11:06:42.244328 140647853295424 tf_logging.py:115] global step 138530: loss = 5.2648 (0.469 sec/step)
INFO:tensorflow:global step 138531: loss = 4.3351 (0.539 sec/step)
I1224 11:06:42.785132 140647853295424 tf_logging.py:115] global step 138531: loss = 4.3351 (0.539 sec/step)
INFO:tensorflow:global step 138532: loss = 6.6549 (0.493 sec/step)
I1224 11:06:43.279867 140647853295424 tf_logging.py:115] global step 138532: loss = 6.6549 (0.493 sec/step)
INFO:tensorflow:global step 138533: loss = 4.9407 (0.493 sec/step)
I1224 11:06:43.774627 140647853295424 tf_logging.py:115] global step 138533: loss = 4.9407 (0.493 sec/step)
INFO:tensorflow:global step 138534: loss = 6.2486 (0.457 sec/step)
I1224 11:06:44.233961 140647853295424 tf_logging.py:115] global step 138534: loss = 6.2486 (0.457 sec/step)
INFO:tensorflow:global step 138535: loss = 4.5404 (0.497 sec/step)
I1224 11:06:44.732757 140647853295424 tf_logging.py:115] global step 138535: loss = 4.5404 (0.497 sec/step)
INFO:tensorflow:global step 138536: loss = 4.8601 (0.450 sec/step)
I1224 11:06:45.184577 140647853295424 tf_logging.py:115] global step 138536: loss = 4.8601 (0.450 sec/step)
INFO:tensorflow:global step 138537: loss = 3.4153 (0.474 sec/step)
I1224 11:06:45.660963 140647853295424 tf_logging.py:115] global step 138537: loss = 3.4153 (0.474 sec/step)
INFO:tensorflow:global step 138538: loss = 3.2380 (0.461 sec/step)
I1224 11:06:46.124583 140647853295424 tf_logging.py:115] global step 138538:

but  after test the model with input 900*9000, its f1-score is lower then the 300*300 model , i can not understand why?

**Comment by dexception** [comment]
I am also facing the same issue even after 200,000 is not going below 1.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
